src/api/comments.ts

```diff
@@ -0,0 +1,37 @@
+export const BASE_URL = 'https://mate.academy/students-api/';
```

Comment:
we can try to use it like that per every api file

```suggestion
export const BASE_URL_COMMENTS = `${process.env.BASE_URL}/comments`;
```